### PR TITLE
Add flag to toggle if docker command will run under root privilege.

### PR DESCRIPTION
**Pull request by borgnix**
Fix issue #72 

Tested on Arch.

### CODE REVIEW

File: docker-process.el [hunk 1]
```diff
@@ -26,16 +26,22 @@
 (require 's)
 (require 'dash)
 
+(defcustom docker-use-root-privilege nil
+  "Boolean value indicating if the docker command will run under root privilege."
```
Comment (Silex) [inline docker-process.el]: `docker-run-as-root` is a better name no?

Also, the docstring should be `Controls wether docker runs as root or not.`

Comment (borgnix) [reply docker-process.el]: Certainly :). I am not very good at naming variable and writing docstring.

**Comment by Silex** [comment]
Alright, please squash the commits as one commit and we are good to go.

**Comment by borgnix** [comment]
Done.

**Comment by Silex** [comment]
Thanks!

**Pull request merged**